pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -217,7 +217,8 @@ func (p *cadvisorStatsProvider) ListPodCPUAndMemoryStats() ([]statsapi.PodStats,
 	return result, nil
 }
 
-func calcEphemeralStorage(containers []statsapi.ContainerStats, volumes []statsapi.VolumeStats, rootFsInfo *cadvisorapiv2.FsInfo) *statsapi.FsStats {
+func calcEphemeralStorage(containers []statsapi.ContainerStats, volumes []statsapi.VolumeStats, rootFsInfo *cadvisorapiv2.FsInfo,
+	podLogStats *statsapi.FsStats, isCRIStatsProvider bool) *statsapi.FsStats {
```

Comment:
maybe I am missing something, but shouldn't `isCRIStatsProvider` always be `false` in `cadvisor_stats_provider.go`?

Reply:
The `calcEphemeralStorage` function is used by both `cadvisor_stats_provider.go` are being reused by `cri_stats_provider.go`.

Moved it into helper.go

Reply:
ah, that makes more sense